model/scheduler_stats_test.go

```diff
@@ -448,5 +448,7 @@ func TestAverageTaskLatencyLastMinuteByDistro(t *testing.T) {
 			AverageTime: 20 * time.Second,
 		},
 	}
-	assert.Equal(expected, latencies.Times)
+	for _, time := range expected {
+		assert.Contains(latencies.Times, time)
```

Comment:
require

Reply:
I just changed this to not depend on the order of the slice. I don't usually use require unless it's something that absolutely must succeed for the test to continue.

Reply:
that's fair.

I tend to use require in loops because i often find that printing dozens (and sometimes 1000s) of lines of spew about the same error is annoying.